src/main/java/me/botsko/prism/database/PrismDataSourceUpdater.java

```diff
@@ -18,5 +18,6 @@
 
     void v6_to_v7();
 
+    void v7_to_v8();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'v7_to_v8' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5609620)